docs/game_and_server_management/Vanity_Perks.md

```diff
@@ -0,0 +1,32 @@
+# Vanity Perks
```

Comment:
I think this got brought back from the dead. It should be removed. https://github.com/discord/discord-api-docs/commit/74660779261e44142120bae0786213ee87232372

But I'll just merge this and remove it for you. XD